ev3dev/core.py

```diff
@@ -942,6 +942,23 @@ def list_motors(name_pattern=Motor.SYSTEM_DEVICE_NAME_CONVENTION, **kwargs):
     return (Motor(name_pattern=name, name_exact=True)
             for name in list_device_names(classpath, name_pattern, **kwargs))
 
+def list_sensors(name_pattern=Sensor.SYSTEM_DEVICE_NAME_CONVENTION, **kwargs):
+    """
+    This is a generator function that enumerates all sensors that match the
+    provided arguments.
+
+    Parameters:
+	name_pattern: pattern that device name should match.
+	    For example, 'sensor*'. Default value: '*'.
+	keyword arguments: used for matching the corresponding device
+	    attributes. For example, driver_name='lego-ev3-touch', or
+	    address=['in1', 'in3']. When argument value is a list, 
+        then a match against any entry of the list is enough.
+    """
+    classpath = abspath(Device.DEVICE_ROOT_PATH + '/' + Sensor.SYSTEM_CLASS_NAME)
```

Comment:
Can you rename this variable (and the other equivalent one) to `class_path`? The current one confuses me because it reads like Java's _classpath_.